package.json

```diff
@@ -17,11 +17,13 @@
     "lint:js": "eslint --fix .",
     "lint:md": "remark -qf .",
     "precommit": "lint-staged",
-    "commitmsg": "commitlint -E GIT_PARAMS"
+    "commitmsg": "commitlint -E GIT_PARAMS",
```

Comment:
GIT_PARAMS or HUSKY_GIT_PARAMS ?

https://github.com/marionebl/commitlint/issues/366

Reply:
Thanks to point this out.

I've simply followed the **Getting started** part of the [README.md](https://github.com/marionebl/commitlint#getting-started).

After some research it seems that both are acceptable related to the comments: https://github.com/marionebl/commitlint/blob/d636811b4537904231c483a05e91aaf96deb7f6e/%40commitlint/cli/src/cli.js#L211-L216.

Reply:
About **husky** and the issue you are mentioned, yes we will have to do the change if we upgrade to the next major version.

Currently we use the version `^0.14.3`.

ref: https://github.com/jleveugle/ovh-manager/blob/sms/package.json#L25

Reply:
Yes, but i think we should use the next major, it's already in RC, and our project is not going in production next week, better to be up-to-date I think.

What do you think about this ?

Reply:
Bonus: Babel 7 is out! 👍

Reply:
Reminder: This pull request is about `conventional-changelog`:) Let's move on!